lib/lambdas/ssmPublisher/handler.ts

```diff
@@ -0,0 +1,65 @@
+import * as lambda from 'aws-lambda';
+import * as log from 'lambda-log';
+import { SsmGateway } from './ssm-param-gateway';
+
+/** Exports here are used by the cdk infrastructure */
+export const SLEEP_SECONDS = 'SLEEP_SECONDS';
+
+// const PHYSICAL_ID = '6db73885-e1cf-4510-a926-31d058538016';
+/**
+
+/**
+ * Entrypoint for the cloudformation custom resource.
+ */
+export async function handler(event: lambda.CloudFormationCustomResourceEvent): Promise<string> {
+
+    log.options.debug = true;
+    log.info('event', { event, envVars: process.env });
+
+    if (!event.RequestType) {
+        const err = `event.RequestType not found. Did you mean to use the custom resource handler?`
+        throw new Error(err);
+    }
+
+    if (!process.env[SLEEP_SECONDS]) throw Error("SLEEP_SECONDS parameter not found.")
+
+    try {
+        // uncomment to spread out the load on SSM a little further.
+        // const sleepSec: number = parseInt(process.env[SLEEP_SECONDS])
+        // await sleep(sleepSec);
+
+        const paramName = event.ResourceProperties['PARAM_NAME']
+        const paramValue = event.ResourceProperties['PARAM_VALUE']
+        console.log(event.ResourceProperties)
+
+        if(!paramName) throw Error(`Expected to find env var PARAM_NAME but did not.`);
+        if(!paramValue) throw Error(`Expected to find env var PARAM_VALUE but did not.`);
```

Comment:
You were saying that the value `0` is being interpreted as null. Perhaps it would be worth changing this check to 
```
if (paramValue === undefined || paramValue === null)
```
to more explicitly catch the case where the parameter has no specified value

Reply:
It actually can't be null due to other workflows in parsing.  However that's probably a good check. 

The issue was that it was treating the number zero as falsy at the construct layer.  So the fix was to ensure they're all strings.